Add missing operators

smkhalsa opened the pull request:
Closes #268
Closes #266

smkhalsa commented:
> Thanks! Can you try adding some tests for this? We'll also need to add an entry in the CHANGELOG and bump the version in the pubspec.yaml

@natebosch done

smkhalsa commented:
@natebosch @kevmoo can we get this merged?

kevmoo commented on pubspec.yaml:
```diff
@@ -1,15 +1,15 @@
 
 environment:
-  sdk: '>=2.6.0 <3.0.0'
+  sdk: ">=2.6.0 <3.0.0"
```
please revert this change and the below change

smkhalsa replied:
Done. I've also reverted this change in https://github.com/dart-lang/code_builder/pull/277

smkhalsa commented:
fixes #268 and #266

smkhalsa commented:
@kevmoo shall I roll this and https://github.com/dart-lang/code_builder/pull/277 into a single PR as 3.3.0-dev?

kevmoo commented:
> @kevmoo shall I roll this and #277 into a single PR as 3.3.0-dev?

Do that here! Any time you add new features, you should bump the minor version.

smkhalsa commented:
> Do that here! Any time you add new features, you should bump the minor version.

@kevmoo Ok, I've added those commits, updated the changelog, and bumped the minor version.

smkhalsa commented:
@kevmoo can we publish 3.3.0-dev to pub.dev please?

kevmoo commented:
@natebosch wants to do a follow-up. We shouldn't have merged in the `part` stuff w/ this PR

natebosch commented:
See https://github.com/dart-lang/code_builder/pull/279

natebosch commented:
We'll also need probably a day to let this go through internal testing.

smkhalsa commented:
@natebosch any update on getting this published?

natebosch commented:
I had some test failures to fix up internally and didn't get this synced until yesterday. Things are looking good though so I'll be publishing now.
